### PR TITLE
Change system fee to gas fee

### PR DESCRIPTION
**Pull request by louisliu2048**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: server/start.go [hunk 1]
```diff
@@ -81,7 +82,7 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().String(flagTraceStore, "", "Enable KVStore tracing to an output file")
 	cmd.Flags().String(flagPruning, "syncable", "Pruning strategy: syncable, nothing, everything")
 	cmd.Flags().String(
-		FlagMinGasPrices, "",
+		FlagMinGasPrices, "0.00000001"+types.DefaultBondDenom,
```
Comment (summerpro) [inline server/start.go]: It is best not to modify it here, otherwise the config file will not take effect

Comment (louisliu2048) [reply server/start.go]: ok. Remove it now.

File: client/flags/flags.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ func PostCommands(cmds ...*cobra.Command) []*cobra.Command {
 		c.Flags().Uint64P(FlagAccountNumber, "a", 0, "The account number of the signing account (offline mode only)")
 		c.Flags().Uint64(FlagSequence, 0, "The sequence number of the signing account (offline mode only)")
 		c.Flags().String(FlagMemo, "", "Memo to send along with transaction")
-		c.Flags().String(FlagFees, "", "Fees to pay along with transaction; eg: 1"+types.DefaultBondDenom)
+		c.Flags().String(FlagFees, "0.0125"+types.DefaultBondDenom, "Fees to pay along with transaction; eg: 1"+types.DefaultBondDenom)
```
Comment (zhongqiuwood) [inline client/flags/flags.go:94]: default value of FlagFees is not expected

File: server/start.go [hunk 3]
```diff
@@ -80,10 +80,10 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().String(flagAddress, "tcp://0.0.0.0:26658", "Listen address")
 	cmd.Flags().String(flagTraceStore, "", "Enable KVStore tracing to an output file")
 	cmd.Flags().String(flagPruning, "syncable", "Pruning strategy: syncable, nothing, everything")
-	cmd.Flags().String(
```
Comment (zhongqiuwood) [inline server/start.go:83]: should keep FlagMinGasPrices with no change here

File: x/auth/types/txbuilder.go [hunk 4]
```diff
@@ -192,7 +192,8 @@ func (bldr TxBuilder) BuildSignMsg(msgs []sdk.Msg) (StdSignMsg, error) {
 		fees = make(sdk.Coins, len(bldr.gasPrices))
 		for i, gp := range bldr.gasPrices {
 			fee := gp.Amount.Mul(glDec)
-			fees[i] = sdk.NewCoin(gp.Denom, fee.Ceil().RoundInt())
+			//fees[i] = sdk.NewCoin(gp.Denom, fee.Ceil().RoundInt())
```
Comment (zhongqiuwood) [inline x/auth/types/txbuilder.go:195]: remove this line

**Pull request merged**